logging/thirdparty/src/test/java/io/quarkus/ts/logging/jboss/SyslogIT.java

```diff
@@ -56,13 +63,26 @@ public void logBigMessage() {
     @Test
     @Tag("https://issues.redhat.com/browse/QUARKUS-4531")
     public void filterBigMessage() {
-        String longerMessage = "Message, which is very long and is not expected to fit into 64 bytes";
+        String shorterMessage = "You won't believe it, but this message is exactly 64 chars long!";
+        String longerMessage = shorterMessage.concat(shorterMessage);
```

Comment:
I think this is absolutely fine. But if it were me, I'd use `shorterMessage.repeat(2)`.

Reply:
Good idea, thank you!